client/README.md

```diff
@@ -0,0 +1,41 @@
+## Mongo Display Setup
```

Comment:
Do we need this readme as well?

Reply:
It details the specific steps for running the frontend / solving any possible bugs you might encounter. We could add this to the root README, though the latter might get really long.